db/migrate/20210813061906_create_call_results.rb

```diff
@@ -0,0 +1,16 @@
+class CreateCallResults < ActiveRecord::Migration[5.2]
+  def change
+    create_table :call_results, id: :uuid do |t|
+      t.uuid :user_id, null: false, foreign_key: true
+      t.uuid :appointment_id, null: false, foreign_key: true
+
+      t.string :cancel_reason
```

Comment:
Should this be required?

Reply:
@rsanheim restricting null at the db + having it optional at the model level is how we're able to sync resources out of order and still enforce presence, this is true of all our sync resources.

`cancel_reason` is not a required field, only applies to a subset of `CallResults`